### DISCUSSION TITLE
[Nix] packaging

### DISCUSSION THREAD

**Discussion by aquifolly** [mdgaziur/findex#38] [Ideas]
It would be nice if there was a nix package

**Comment by mdgaziur** [comment]
Unfortunately I never used NIX. Any contributions are welcome!

**Reply by aquifolly** [reply]
changed my mind, a regular nix package will do

**Reply by aquifolly** [reply]
would you like to be a maintainer of the package?

**Reply by mdgaziur** [reply]
Short answer is no.

Long answer: actually I'm not gonna be able to be a maintainer of anything for the next 12-15 months due to having an extremely short time to finish whole college syllabus. Probably I'll be back on track in Aug or September of next year. There might be small updates now and then. But I am not gonna be able to do proper "maintenance" on packages. If you want, you can maintain the nix package :)

**Reply by aquifolly** [reply]
Ok

**Reply by aquifolly** [reply]
unfortunately, I am in a similar situation, so we might have to rely on the rest of the nix community to keep it up to date, but I could be the primary maintainer

**Reply by mdgaziur** [reply]
Alright. If you need any help, let me know. :)

**Comment by aquifolly** [comment]
It has been merged

**Reply by aquifolly** [reply]
but I need to update it to the latest release

**Reply by mdgaziur** [reply]
That's really cool!

**Reply by aquifolly** [reply]
the build fails for some reason
https://github.com/NixOS/nixpkgs/pull/219032/checks?check_run_id=11696230719
do you happen to know why?

**Comment by mdgaziur** [comment]
Ahh, I didn't commit updated cargo.lock. I'll fix this soon.
